classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -467,6 +467,20 @@ function reviewRead($args, $request) {
 			$reviewAssignmentDao->updateObject($reviewAssignment);
 		}
 
+		if (!$reviewAssignment->getDateCompleted()) {
```

Comment:
Well, I did it like that: the editor is using the same button/function to mark an review as completed and also read. I could add a new function but I believe it is not necessary -- maybe we can see it during the presentation on Monday...